### DISCUSSION TITLE
Servers in OpenAPI spec have a trailing slash which is against Spectral rules

### DISCUSSION THREAD

**Discussion by jimbobbennett** [fastapi/fastapi#10309] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
app = FastAPI(
    servers=[
        {
            "url": "http://localhost:8000",
            "description": "Prod"
        }
    ],
)
```

### Description

When you set a server when declaring the `FastAPI` object, this gets added to the OpenAPI spec with a trailing slash:

```json
"servers": [
    {
      "url": "http://localhost:8000/",
      "description": "Prod"
    }
  ],
```

If you run the spectral analyzer against this, it gives warning [oas3-server-trailing-slash](https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#oas3-server-trailing-slash).

Looking at the examples in the FastAPI docs, none of the examples have trailing slashes: https://fastapi.tiangolo.com/advanced/behind-a-proxy/?h=servers+openapi#additional-servers.

Is there a way to generate servers without a trailing slash?

### Operating System

Linux, macOS

### FastAPI Version

0.103.1

### Pydantic Version

2.3.0

### Python Version

Python 3.9.16

**Comment by YuriiMotov** [comment]
Fixed
